tools/python_api/src_py/connection.py

```diff
@@ -118,3 +118,15 @@ def _get_node_property_names(self, table_name):
                 "is_primary_key": is_primary_key
             }
         return results
+
+    def set_query_timeout(self, timeout_in_ms):
```

Comment:
Doc for this function should also be added to the beginning of the class.

Reply:
done